<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>199</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Your Monthly Budget</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setBudgetButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>199</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Budget</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="budgetLineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>199</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>199</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Expense</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="expenseLineEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>199</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="categoryComboBox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>199</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Food</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Entertainment</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transportation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Housing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Health &amp; Fitness</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Education</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Savings</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Etc</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addExpenseButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>199</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Expense</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>199</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Expenses</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="expenseTableWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>category</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="viewReportButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>199</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>view Report</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>450</y>
      <width>199</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string> Budget Status: $0.00</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>49</y>
      <width>251</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
